source/module_io/output_sk.h

```diff
@@ -0,0 +1,33 @@
+#ifndef OUTPUT_SK_H
+#define OUTPUT_SK_H
+
+#include "module_basis/module_ao/parallel_orbitals.h"
+#include "module_hamilt_lcao/hamilt_lcaodft/hamilt_lcao.h"
+#include "module_io/output_interface.h"
+
+namespace ModuleIO
+{
+
+template <typename TK>
+class Output_Sk : public Output_Interface
```

Comment:
What is SK stands for?

Reply:
SK stands for overlap matrix in the reciprocal space